offline-runtime/api/src/com/intellij/rt/coverage/offline/api/CoverageRuntime.java

```diff
@@ -43,6 +43,11 @@ public static List<ClassCoverage> collectClassfileData(final List<byte[]> classF
     return CoverageCollector.collect(projectData, classFinder);
   }
 
+  public static void dumpIcReport(DataOutput output, File errorFile) throws IOException {
```

Comment:
Do I get it right that this method is an "API" method that will be caused by kover?

Reply:
Yes, this API is universal, with the help of it I can save to a file and to an byte array